externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/GetFunctions.scala

```diff
@@ -54,6 +55,17 @@ class GetFunctions(
         " function within its schema")
   }
 
+  override protected def beforeRun(): Unit = {
```

Comment:
can we keep the meta operations unchanged for now

Reply:
ok